dashboard/src/types/tree/BuildDetails.tsx

```diff
@@ -0,0 +1,29 @@
+export type TBuildDetails = {
+  timestamp: string;
+  id: string;
+  architecture?: string;
+  checkout_id?: string;
+  command?: string;
+  comment?: string;
+  config_name?: string;
+  valid?: boolean;
+  start_time?: string;
+  duration?: string;
+  compiler?: string;
+  config_url?: string;
+  log_url?: string;
+  tree_name?: string;
+  git_commit_hash?: string;
+  git_commit_name?: string;
+  git_repository_branch?: string;
+  git_repository_url?: string;
+  origin?: string;
+  misc: IBuildDetailsMisc | null;
+};
+
+interface IBuildDetailsMisc {
+  kernel_type?: string;
+  dtb?: string;
+  modules?: string;
+  system_map?: string;
+}
```

Comment:
how are you defining this type? it should be generated from open api standard

at least add a ticket for it

`https://www.django-rest-framework.org/topics/documenting-your-api/#drf-spectacular`